deploy/aws/app/kube/main.tf

```diff
@@ -12,7 +12,7 @@ module "app_label" {
 
 module "app" {
 
-  source = "git::https://github.com/amido/stacks-terraform//aws/modules/infrastructure_modules/stacks_app?ref=master"
+  source = "git::https://github.com/ensono/stacks-terraform//aws/modules/infrastructure_modules/stacks_app?ref=master"
```

Comment:
Could you review this URL? It doesn't seem to work either

Reply:
the amido one doesn't work either